galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java

```diff
@@ -21,12 +21,12 @@
  * 
  * @galasa.required No
  * 
- * @galasa.default docker.galasa.dev
+ * @galasa.default harbor.galasa.dev/docker_proxy_cache
  * 
  * @galasa.valid_values a valid hostname
  * 
  * @galasa.examples 
- * <code>galasaecosystem.docker.version=docker.galasa.dev</code>
+ * <code>galasaecosystem.docker.version=harbor.galasa.dev/docker_proxy_cache</code>
```

Comment:
This @galasa.examples line also contains the wrong key for the CPS property: it should say galasaecosystem.docker.registry. Please can you update?